buildenv/jenkins/JenkinsfileBase

```diff
@@ -474,13 +476,14 @@ def get_sources() {
 }
 
 def make_test() {
+	String makeTestCmd = "if [ `uname` = AIX ] || [ `uname` = SunOS ] || [ `uname` = *BSD ]; then MAKE=gmake; else MAKE=make; fi; cd ./openjdk-tests/TKG; \$MAKE compile"
```

Comment:
Instead of having this defined multiple times, can we have one variable for it?

Reply:
Hi @llxia , 

Yes, I've updated the PR based on your suggestion. We now have it as a variable. 

Retested: 
Internal/Grinder/13872
Internal/Grinder/13874